controller/src/main/java/io/pravega/controller/store/stream/StreamMetadataStore.java

```diff
@@ -202,6 +234,23 @@
     CompletableFuture<List<Integer>> getActiveSegments(final String scope, final String name, final long timestamp, final OperationContext context, final Executor executor);
```

Comment:
Please change the return type of this method to `CompletableFuture<List<Segment>>` so that it is consistent with other overloads of this method.